### ISSUE TITLE
Help with fine tuning?

### ISSUE THREAD

**Issue by ectg** [bertinetto/cfnet#33]
Hi Luca. Thanks for your work!

I would like to fine tune your pre-trained Imagenet weights using my smallish video data. 

The object I need to track is not one of the classes in Imagenet. I used [this](https://github.com/www0wwwjs1/tensorflow-siamese-fc) implementation out of the box, it works fairly well, but I would like to improve the accuracy by training/tuning on my own data. I was wondering approximately how many new videos and frame pairs would I need to get good results. ?

**Comment by bertinetto** [comment]
Hi!
We always wanted to try, but we never actually did it.
It would be interesting for example to see how it works for pedestrians or faces, both classes that aren't in VID and for which there is large abundance.
I am not sure what are the hyperparameters for the LR (e.g. starting point and schedule) that would work best though.

Don't hesitate to let us know if you make any progress on this.

Cheers,
Luca
